system/jlib/jutil.cpp

```diff
@@ -3123,6 +3123,10 @@ static StringBuffer &doGetTempFilePath(StringBuffer & target, const char *tempCa
     StringBuffer dir;
     if (pTree)
         getConfigurationDirectory(pTree->queryPropTree("Directories"),tempCategory,component,pTree->queryProp("@name"),dir);
+#ifdef _CONTAINERIZED
```

Comment:
@jakesmith please check this change in particular.

Reply:
@ghalliday I think change is fine, but it's never going to use the pTree even if passed in in containerized mode, so I think I'd have coded change as:
```
    IPropertyTree *dirTree = nullptr;
#ifndef _CONTAINERIZED
    if (pTree)
        dirTree = pTree->queryPropTree("Directories");
#endif
    getConfigurationDirectory(dirTree,tempCategory,component,nullptr,dir);
```